install_scripts/install_lago.sh

```diff
@@ -39,13 +39,30 @@ install_lago() {
 }
 
 add_lago_repo() {
+    local DISTRO
     if ! grep -q "Fedora" /etc/redhat-release; then
-        yum -y install epel-release
-        local DISTRO=el
+        DISTRO="el"
+        if grep -q "CentOS" /etc/redhat-release; then
+            yum -y install epel-release
+        else
+            echo "
+            Detected distro is RHEL, please ensure you have the following
+            subscriptions enabled:
+            rhel-7-server-rpms
+            rhel-7-server-rhv-4-mgmt-agent-rpms
+            EPEL
```

Comment:
They can be in some RHEL channels, or in CentOS.
If EPEL is needed - bring the relevant RPM - or fix the documentation on how to do it.